tailord/src/profiles.rs

```diff
@@ -124,7 +134,7 @@ impl Profile {
         let link = std::fs::read_link(ACTIVE_PROFILE_PATH)
             .map_err(|err| fdo::Error::IOError(err.to_string()))?;
         let components: Vec<Component> = link.components().collect();
-        if components.len() == 2 {
+        if components.len() > 0 {
```

Comment:
Was this `== 2` check intentional? It doesn't seem necessary.
I had to modify this, otherwise it failed to read the default profile.

Reply:
I think the shortest relative path should always have exactly 2 segments and so far, tailord only used relative paths. If this check works for you and since using the absolute path for the active profile should probably work too, I think this is a good change.